trace/google/cloud/trace/_gax.py

```diff
@@ -0,0 +1,259 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""GAX Wrapper for interacting with the Stackdriver Trace API."""
+
+from google.cloud.gapic.trace.v1.trace_service_client import (
```

Comment:
> an we enforce this going forward?

IIUC @lukesneeringer (and I) would prefer lint rules that can be enforced by tools, not humans